dev/ParallaxView/ScrollInputHelper.h

```diff
@@ -29,8 +29,8 @@ class ScrollInputHelper
     static winrt::RichEditBox GetRichEditBoxParent(const winrt::DependencyObject& childElement);
     static void GetChildScrollerOrScrollViewer(
         const winrt::DependencyObject& rootElement,
-        _Out_ winrt::Scroller* scroller,
-        _Out_ winrt::FxScrollViewer* scrollViewer);
+        winrt::Scroller* scroller,
+        winrt::FxScrollViewer* scrollViewer);
     winrt::UIElement GetScrollContentElement() const;
```

Comment:
Recommend: modernize this function to return as tuple (or put back _Out_ for now and make @RBrid do it when he's back)